lib/yabeda/metric.rb

```diff
@@ -14,6 +14,7 @@ class Metric
     option :per,     optional: true, comment: "Per which unit is measured `unit`. E.g. `call` as in seconds per call"
     option :group,   optional: true, comment: "Category name for grouping metrics"
     option :aggregation, optional: true, comment: "How adapters should aggregate values from different processes"
+    option :adapter, optional: true, comment: "Which adapter apply the indicator settings."
```

Comment:
Let's reword comment a bit for clarity:
```suggestion
    option :adapter, optional: true, comment: "Monitoring system adapter to register metric in and report metric values to (other adapters won't be used)"
```